pkg/filter/stream/faulttolerance/regulator/measure_model.go

```diff
@@ -91,7 +91,8 @@ func (m *MeasureModel) snapshotInvocations(recoverTime int64) []*InvocationStat
 			return true
 		}
 		if stat.GetCall() <= 0 {
-			if stat.AddUselessCycle() {
+			// don't release unhealth stat host
+			if stat.AddUselessCycle() && stat.IsHealthy() {
```

Comment:
there are already unhealthy judgments in front of this cycle. And with one, the semantics are more complete.